src/actions/action-creators.test.js

```diff
@@ -126,7 +128,147 @@ describe('Action creators', () => {
     });
   });
 
+  /***** async *****/
   describe('async action creators', () => {
-    // TODO
+    let store;
+    beforeEach(() => {
+      store = mockStore({
+        images: {
+          frames: {},
+          frameIDs: [],
+          gifProgress: 0,
+          gifData: ''
+        },
+        settings: {
+          image: {
+            width: 300,
+            height: 300,
+            interval: 100,
+            oversample: false
+          },
+          bounds: {
+            left: -10,
+            right: 10,
+            bottom: -10,
+            top: 10
+          },
+          strategy: 'contain'
+        },
+        ui: {
+          expandedPane: 'NONE',
+          previewIdx: 0,
+          playing: false,
+          error: ''
+        }
+      });
+    });
+
+    // flashError
+    it('return an action to temporarily display error message', () => {
+      jest.useFakeTimers();
+      const expectedActions = [
+        { type: types.SET_ERROR, payload: { message: 'error' } },
+        { type: types.CLEAR_ERROR }
+      ];
+      store.dispatch(actions.flashError('error'));
+      jest.runAllTimers();
+      expect(store.getActions()).toEqual(expectedActions);
+    });
+
+    // requestFrame
+    it('return an action to request a frame from the calculator', () => {
+      const expectedActions = [
+        {
+          type: types.ADD_FRAME,
+          payload: { id: 2, imageData: expect.any(String) }
+        }
+      ];
+      const opts = {
+        height: 300,
+        targetPixelRatio: 1,
+        width: 300
+      };
+      initializeCalculator(
+        // desmos mock
+        {
```

Comment:
can you use the same desmos mock for both tests? would be nice to move this out into a variable called `desmosMock` that you just reference in the tests that need it. the variable name would also reduce the need for the comment a line above.